Integrations/QuinC/Pipfile

```diff
@@ -0,0 +1,23 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pylint = "*"
+pytest = "*"
+pytest-mock = "*"
+requests-mock = "*"
+flake8 = "*"
+
+[packages]
+certifi = "==2017.11.5"
+chardet = "==3.0.4"
+idna = "==2.6"
+olefile = "==0.44"
+requests = "==2.18.4"
+urllib3 = "==1.22"
+PyYAML = "==3.12"
+
+[requires]
+python_version = "2.7"
```

Comment:
you are using python 3.7 not 2.7
change it to 3.7 or copy it from 
You can copy Pipfile and Pipfile.lock from https://github.com/demisto/content/tree/master/Integrations/AttackIQFireDrill